Description: fix Googletest function name
 Old version of Googletest use different (now deprecated) function name.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: not-needed
Last-Update: 2019-08-06

---

--- protobuf-3.9.1.orig/src/google/protobuf/compiler/command_line_interface_unittest.cc
+++ protobuf-3.9.1/src/google/protobuf/compiler/command_line_interface_unittest.cc
@@ -2719,7 +2719,7 @@ TEST_P(EncodeDecodeTest, ProtoParseError
       "net/proto2/internal/no_such_file.proto: No such file or directory\n");
 }
 
-INSTANTIATE_TEST_SUITE_P(FileDescriptorSetSource, EncodeDecodeTest,
+INSTANTIATE_TEST_CASE_P(FileDescriptorSetSource, EncodeDecodeTest,
                          testing::Values(PROTO_PATH, DESCRIPTOR_SET_IN));
 }  // anonymous namespace
 
--- protobuf-3.9.1.orig/src/google/protobuf/descriptor_unittest.cc
+++ protobuf-3.9.1/src/google/protobuf/descriptor_unittest.cc
@@ -2990,7 +2990,7 @@ TEST_P(AllowUnknownDependenciesTest,
   ASSERT_TRUE(pool_->FindMessageTypeByName("undeclared.Quux") == nullptr);
 }
 
-INSTANTIATE_TEST_SUITE_P(DatabaseSource, AllowUnknownDependenciesTest,
+INSTANTIATE_TEST_CASE_P(DatabaseSource, AllowUnknownDependenciesTest,
                          testing::Combine(testing::Values(NO_DATABASE,
                                                           FALLBACK_DATABASE),
                                           testing::Values("proto2", "proto3")));
--- protobuf-3.9.1.orig/src/google/protobuf/dynamic_message_unittest.cc
+++ protobuf-3.9.1/src/google/protobuf/dynamic_message_unittest.cc
@@ -320,7 +320,7 @@ TEST_F(DynamicMessageTest, Proto3) {
   delete message;
 }
 
-INSTANTIATE_TEST_SUITE_P(UseArena, DynamicMessageTest, ::testing::Bool());
+INSTANTIATE_TEST_CASE_P(UseArena, DynamicMessageTest, ::testing::Bool());
 
 }  // namespace protobuf
 }  // namespace google
--- protobuf-3.9.1.orig/src/google/protobuf/map_field_test.cc
+++ protobuf-3.9.1/src/google/protobuf/map_field_test.cc
@@ -295,7 +295,7 @@ class MapFieldStateTest : public testing
   State state_;
 };
 
-INSTANTIATE_TEST_SUITE_P(MapFieldStateTestInstance, MapFieldStateTest,
+INSTANTIATE_TEST_CASE_P(MapFieldStateTestInstance, MapFieldStateTest,
                          ::testing::Values(CLEAN, MAP_DIRTY, REPEATED_DIRTY));
 
 TEST_P(MapFieldStateTest, GetMap) {
--- protobuf-3.9.1.orig/src/google/protobuf/text_format_unittest.cc
+++ protobuf-3.9.1/src/google/protobuf/text_format_unittest.cc
@@ -80,7 +80,7 @@ const std::string kEscapeTestStringEscap
 
 class TextFormatTest : public testing::Test {
  public:
-  static void SetUpTestSuite() {
+  static void SetUpTestCase() {
     GOOGLE_CHECK_OK(File::GetContents(
         TestUtil::GetTestDataPath(
             "net/proto2/internal/"
@@ -103,7 +103,7 @@ std::string TextFormatTest::static_proto
 
 class TextFormatExtensionsTest : public testing::Test {
  public:
-  static void SetUpTestSuite() {
+  static void SetUpTestCase() {
     GOOGLE_CHECK_OK(File::GetContents(
         TestUtil::GetTestDataPath("net/proto2/internal/testdata/"
                                   "text_format_unittest_extensions_data.txt"),
--- protobuf-3.9.1.orig/src/google/protobuf/util/internal/default_value_objectwriter_test.cc
+++ protobuf-3.9.1/src/google/protobuf/util/internal/default_value_objectwriter_test.cc
@@ -73,7 +73,7 @@ class DefaultValueObjectWriterTest : pub
   virtual ~DefaultValueObjectWriterTest() {}
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          DefaultValueObjectWriterTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -160,7 +160,7 @@ class DefaultValueObjectWriterSuppressLi
   ~DefaultValueObjectWriterSuppressListTest() override {}
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          DefaultValueObjectWriterSuppressListTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
--- protobuf-3.9.1.orig/src/google/protobuf/util/internal/protostream_objectsource_test.cc
+++ protobuf-3.9.1/src/google/protobuf/util/internal/protostream_objectsource_test.cc
@@ -290,7 +290,7 @@ class ProtostreamObjectSourceTest
   bool render_unknown_enum_values_;
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtostreamObjectSourceTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -606,7 +606,7 @@ class ProtostreamObjectSourceMapsTest :
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtostreamObjectSourceMapsTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -748,7 +748,7 @@ class ProtostreamObjectSourceAnysTest :
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtostreamObjectSourceAnysTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -985,7 +985,7 @@ class ProtostreamObjectSourceStructTest
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtostreamObjectSourceStructTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -1031,7 +1031,7 @@ class ProtostreamObjectSourceFieldMaskTe
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtostreamObjectSourceFieldMaskTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -1094,7 +1094,7 @@ class ProtostreamObjectSourceTimestampTe
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtostreamObjectSourceTimestampTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
--- protobuf-3.9.1.orig/src/google/protobuf/util/internal/protostream_objectwriter_test.cc
+++ protobuf-3.9.1/src/google/protobuf/util/internal/protostream_objectwriter_test.cc
@@ -187,7 +187,7 @@ class ProtoStreamObjectWriterTest : publ
   virtual ~ProtoStreamObjectWriterTest() {}
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -1194,7 +1194,7 @@ class ProtoStreamObjectWriterTimestampDu
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterTimestampDurationTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -1590,7 +1590,7 @@ class ProtoStreamObjectWriterStructTest
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterStructTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -1781,7 +1781,7 @@ class ProtoStreamObjectWriterMapTest : p
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterMapTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -1878,7 +1878,7 @@ class ProtoStreamObjectWriterAnyTest : p
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterAnyTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -2523,7 +2523,7 @@ class ProtoStreamObjectWriterFieldMaskTe
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterFieldMaskTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -2776,7 +2776,7 @@ class ProtoStreamObjectWriterWrappersTes
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterWrappersTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
@@ -2799,7 +2799,7 @@ class ProtoStreamObjectWriterOneOfsTest
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(DifferentTypeInfoSourceTest,
+INSTANTIATE_TEST_CASE_P(DifferentTypeInfoSourceTest,
                          ProtoStreamObjectWriterOneOfsTest,
                          ::testing::Values(
                              testing::USE_TYPE_RESOLVER));
